src/pages/Login/index.tsx

```diff
@@ -26,10 +26,7 @@ function LoginPage() {
 
     if (location.state?.isPasswordReset) {
       // Shown for 5 seconds
-      showSuccessNotification(t('passwordResetSuccessful'), {
-        duration: 5000,
-        allowDuplicate: false,
-      });
+      addSuccessBanner({ children: t('passwordResetSuccessful'), duration: 5000 });
```

Comment:
`allowDuplicate` is no longer relevant with the new infrastructure as duplicates of the same banner key is impossible.

Reply:
Indeed, that could be desirable in the future, however it would require a new strategy for identifying banners than we currently have in place. Could be that we provide an arbitrary unique `key` string, and separately provide the `*Banner` component that we wish to have rendered, rather than tightly couple the key with the component as we're doing now.

I'd say let's support that if we need it down the road.